lms/djangoapps/courseware/views.py

```diff
@@ -1370,3 +1371,33 @@ def _track_successful_certificate_generation(user_id, course_id):  # pylint: dis
                 }
             }
         )
+
+
+@require_http_methods(["GET", "POST"])
+def render_xblock(request, usage_key_string):
+    """
+    Returns an HttpResponse with HTML content for the xBlock with the given usage_key.
+    The returned HTML is a chromeless rendering of the xBlock (excluding content of the containing courseware).
+    """
+    usage_key = UsageKey.from_string(usage_key_string)
+    usage_key = usage_key.replace(course_key=modulestore().fill_in_run(usage_key.course_key))
+    course_key = usage_key.course_key
+
+    # verify the user has access to the course, including enrollment check
+    course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=True)
+
+    # get the block, which verifies whether the user has access to the block.
+    block, _ = get_module_by_usage_id(request, unicode(course_key), unicode(usage_key), disable_staff_debug_info=True)
```

Comment:
I would have thought it would be convenient, particularly in the LTI use case, since the staff may not interact regularly with the LMS. @mcgachey: any thoughts on this?